Yoast/wordpress-seo
# Introduce meta surface class

## Context
This introduces a new class to make it significantly easier to access all our meta output. This will better enable themes and other plugins, including our add-ons, to integrate with Yoast SEO.

## Summary

This PR can be summarized in the following changelog entry:

* TODO: Write documentation and link to it here.

## Relevant technical choices:

* All presenters have a new optional argument, `$output_tags`. This can be set to false for them to output their raw value instead of the full tags. This allows them to be used to by the meta surface to output values instead of duplicating code.

## Test instructions

This PR can be tested by following these steps:

* If you're using the plugins docker run `./wp.sh shell` to start an interactive WP shell.
* Look into the surface file to see the different functions available for finding indexables.
* Call one of these and then access any property normally available on a presentation.
* You should see the same output you would normally see in the meta tags.
* For example: `YoastSEO()->meta->for_post( 5 )->title;`

## UI changes

## Documentation

## Quality assurance
